identity/src/main/java/com/android/identity/GattClient.java

```diff
@@ -75,14 +77,16 @@ class GattClient extends BluetoothGattCallback {
     private int mNegotiatedMtu;
     private byte[] mIdentValue;
     private boolean mUsingL2CAP = false;
-
-    GattClient(@NonNull Context context, @LoggingFlag int loggingFlags, @NonNull UUID serviceUuid,
-            @NonNull byte[] encodedEDeviceKeyBytes,
-            @NonNull UUID characteristicStateUuid,
-            @NonNull UUID characteristicClient2ServerUuid,
-            @NonNull UUID characteristicServer2ClientUuid,
-            @Nullable UUID characteristicIdentUuid,
-            @Nullable UUID characteristicL2CAPUuid) {
+    private boolean mClearCache;
```

Comment:
consistency nit: want to explicitly default this like the other boolean flag?

alternatively, remove default on the other flag?

Reply:
that's fair